symbols/ram.yml

```diff
@@ -524,6 +576,11 @@ ram:
       address:
         EU: 0x22B9F88
         NA: 0x22B9648
+        JP: 0x22BAE00
+      length:
+        EU: 0xC
```

Comment:
This length is wrong, it's unknown but definitely not 0xC

Reply:
I put 0xC here because there is another symbol at NA 0x022B9654, so the current assumption that all the fields in `thread_info` are part of one struct may be incorrect. Though I don't have more context on this field so I'll remove the length for now.